src/plugins/dashboard/public/application/utils/use/use_dashboard_container.tsx

```diff
@@ -4,12 +4,21 @@
  */
 
 import React, { useState } from 'react';
-import { EMPTY, Subscription, merge } from 'rxjs';
-import { catchError, distinctUntilChanged, map, mapTo, startWith, switchMap } from 'rxjs/operators';
+import _, { uniqBy } from 'lodash';
```

Comment:
i think we are in the process of removing lodash right?

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3653

Looks like this issue is tracking the process of removing lodash. Dashboard plugin previously has 16 usages of lodash, and it seems like the process of removing lodash from dashboard plugin has not begun. Dashboard de-angular will not introduce any new usages of lodash.